src/main/java/pl/miloszkajetan/serwisAukcyjny/Data/UsersRepository.java

```diff
@@ -0,0 +1,14 @@
+package pl.miloszkajetan.serwisAukcyjny.Data;
+
+import org.apache.catalina.User;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.Optional;
+
+public interface UsersRepository <T extends User> extends JpaRepository <T, Long> { //Zapytać Olka o <T extends User>
```

Comment:
public interface UsersRepository extends JpaRepository <User, Long> 
To jest poprawny zapis